syft/execution/plan.py

```diff
@@ -422,7 +433,12 @@ def __call__(self, *args, **kwargs):
                 if _self is None:
                     response = eval(cmd)(*args, **kwargs)  # nosec
                 else:
-                    response = getattr(_self, cmd)(*args, **kwargs)
+                    try:
+                        response = getattr(_self, cmd)(*args, **kwargs)
```

Comment:
Q: What is the better method to do it?
The problem is that when running the plan and we reach the ```get_plain_text``` instruction, ```_self``` is a ```PlaceHolder``` and ```.child``` is an ```MPCTensor```. The placeholder does not have the ```get_plain_text``` method and in the current implementation I try to resolve it by checking the child

Reply:
If `Placeholder` isn't properly forwarding to the `child`, we should probably fix that in `Placeholder`.